tools/ci/check_grep.sh

```diff
@@ -173,6 +173,14 @@ do
     done < <(jq -r '[.map_file] | flatten | .[]' $json)
 done
 
+part "updatepaths validity"
+lines=$(find tools/UpdatePaths/Scripts -type f ! -name "*.txt" | wc -l)
+if [ $lines -gt 0 ]; then
+	echo
+	echo -e "${RED}ERROR: Found an UpdatePaths File that doesn't end in .txt! Please add the proper file extension!${NC}"
+	st=1
```

Comment:
It's inconsistently done but it seems spaces is more used than tabs so I guess we'll stick to that

```suggestion
    echo
    echo -e "${RED}ERROR: Found an UpdatePaths File that doesn't end in .txt! Please add the proper file extension!${NC}"
    st=1
```